src/locales/en-US/network.js

```diff
@@ -28,5 +24,5 @@ export default {
   'network.hint.uniqueName': '{name} has been used! Try a new name.',
   'network.hint.selectNode': 'Select a node',
   'network.hint.selectInterfaces': 'Select interfaces',
-  'network.hint.VLANTag': 'Please enter a number between 0 ~ 4095.',
-}
+  'network.hint.vlanTags': 'Please enter a number between 0 ~ 4095.',
```

Comment:
The whole name of VLAN is Virtual Local Area Network.
Not sure that convert VLAN to lower case is good?
What do you think?
Thanks.

Reply:
keep `VLANTags` and I have to put the missed s back to source code.